spec/models/schedule_spec.rb

```diff
@@ -3,17 +3,48 @@
 RSpec.describe Schedule, type: :model do
   describe 'validation' do
     context '#from' do
-      it 'is valid with valid attributes' do
-        expect(build(:schedule, from: 6.weeks.from_now.beginning_of_day)).to be_valid
-      end
+      context 'first schedule' do
+        let :user do
```

Comment:
Nit-pick but can we wrap the `let` arguments in parens? This seems to be the prevalant style.